utils/fastmath/dev/ggplot.clj

```diff
@@ -238,3 +264,92 @@
       (title (r.base/expression (symbol "ABC~x^2~frac(1,2)~sqrt(2,3)")))
       (->file))
 
+
+(defn fgraph-int
+  ([f domain]
+   (fgraph-int f domain {:title ""}))
+  ([f domain {:keys [title]}]
+   (function f {:x domain
+                :title title})))
+
+
+(defn sample-int
+  [f ^long dx ^long dy]
+  (map (fn [v] [(+ v 0.5) (f v)]) (range dx (inc dy))))
+
+
+(defn bgraph-int
+  ([f] (bgraph-int f nil))
+  ([f domain] (bgraph-int f domain {:title ""}))
+  ([f domain opts]
+   (let [[dx dy] domain
+         xsys (if (map? f)
+                (seq f)
+                (sample-int f dx dy))
+         xs (map first xsys)
+         ys (map second xsys)]
+     (lollipop xs ys opts))))
+
+
+(defn dgraph-cont
+  "params:
+  `opts`:
+     keys:`:pdf` a vector of pdf range, default 0 to 1
+          `:icdf` a vector of icdf range, default 0 0.999
+          `:data` supplied data"
+  ([pdf-graph distr] (dgraph-cont pdf-graph distr nil))
+  ([pdf-graph distr {:keys [pdf icdf data]
+                     :or {pdf [0 1] icdf [0 0.999]}}]
+   (let [pdf-plot (pdf-graph (if data data (partial fr/pdf distr)) pdf {:title "PDF"})
+         cdf-plot (fgraph-int (partial fr/cdf distr) pdf {:title "CDF"})
+         icdf-plot (fgraph-int (partial fr/icdf distr) icdf {:title "ICDF"})]
+     (->image
+      #_{:clj-kondo/ignore [:unresolved-namespace]}
+      (pw/wrap_plots pdf-plot cdf-plot icdf-plot :ncol 3)))))
```

Comment:
Maybe instead of packing three charts into one, let's do three separate charts and pack them into table. This way KDE table will look better.

Reply:
Yes, now it returns vector of charts.